init_db.py

```diff
@@ -0,0 +1,100 @@
+# Initialize database and populate initial values
+
+from databases import Database
+
+import asyncio
+import json
+
+database = Database('sqlite+aiosqlite:///wordle_app.db')
+
+async def init_db():
+    await database.connect()
+
+    query = "DROP TABLE IF EXISTS user"
+    await database.execute(query=query)
+    query = "DROP TABLE IF EXISTS games"
+    await database.execute(query=query)
+    query = "DROP TABLE IF EXISTS guesses"
+    await database.execute(query=query)
+    query = "DROP TABLE IF EXISTS secret_word"
+    await database.execute(query=query)
+    query = "DROP TABLE IF EXISTS valid_words"
+    await database.execute(query=query)
+    
+
+    query = """
+            CREATE TABLE user (
+                userid INTEGER NOT NULL PRIMARY KEY ASC, 
```

Comment:
```AUTOINCREMENT``` does not reuse ```row_id``` while this current set up does which means we cannot delete user record from this  database since that may cause a new user having the id of a deleted user

@kencue propose to keep this set up but add an ```is_active``` column in the future there is a request to handle deleted users to prevent issue above